crates/holochain_sqlite/src/sql/p2p_agent_store/query_near_basis.sql

```diff
@@ -0,0 +1,40 @@
+-- simple select the whole table
+SELECT encoded FROM (
+  SELECT
+    encoded AS encoded,
+    -- if we have two spans
+    CASE WHEN storage_start_1 IS NOT NULL
+      AND storage_end_1 IS NOT NULL
+      AND storage_start_2 IS NOT NULL
+      AND storage_end_2 IS NOT NULL
```

Comment:
I wish my PR that simplified arc queries down to a single range had already been merged :_(

Reply:
I guess I will wait to merge this until after this PR is merged https://github.com/holochain/holochain/pull/816